Tests/Unit/Backend/SolrModule/TikaControlPanelModuleControllerTest.php

```diff
@@ -63,17 +63,20 @@ public function canShowInformationFromStandaloneTikaServer()
 
         $this->controller->setTikaService($tikaServerService);
         $this->controller->setTikaConfiguration(
-            array('extractor' => 'server', 'tikaServerPath' => $this->getFixturePath('fake-server-jar.jar'))
+            [
+                'extractor' => 'server',
+                'tikaServerPath' => $this->getFixturePath('fake-server-jar.jar')
+            ]
         );
 
         $this->viewMock->expects($this->at(2))->method('assign')->with('server',
```

Comment:
Here, I'd actually move `'server'` to the next line, have it align with the opening square bracket of the array